tools/importers/SynchronizeTypographyTokens.ts

```diff
@@ -79,18 +84,22 @@ export const ImportTypographyVariables = (
 						propertyValue = propertyOptions[variableKey].value;
 						break;
 				}
+
 				if (propertyValue !== undefined) {
 					resultCss.push('--' + propertyName + ': ' + propertyValue + ';');
 					resultScss.push('$' + propertyName + ': var(--' + propertyName + ');');
 
+					const propertyNameSplit = propertyName.split('-');
 					const resultJsonObject: ITokenJsonObject = {
 						id: (
 							typographyConfigFile.bin.files.tokens.destinationJson +
 							'_' +
 							propertyName
-						).replace(/[^a-zA-Z ]/g, ''),
-						label: propertyName,
+						).replace(/[^a-zA-Z0-9 ]/g, ''),
+						label: propertyNameSplit.slice(-1).pop() + '',
```

Comment:
```suggestion
						label: propertyNameSplit.slice(-1).pop() || '',
```

This syntax makes it clear that we're handling falsy value, not casting from other type

Reply:
Ok, changed.